parsers/nmparser/get_model_output.go

```diff
@@ -107,6 +107,14 @@ func GetModelOutput(lstPath string, ext ModelOutputFile, grd bool, shk bool) (Su
 
 		// Parse parameters from .ext
 		extData, parameterNames := ParseParamsExt(extLinesParsed)
+		lenData := len(extData)
+		lenMethods := len(results.RunDetails.EstimationMethods)
+		if lenData != lenMethods {
```

Comment:
There is one estimation method that doesn't generate parameter estimates (`METHOD=CHAIN`); this is used with `$EST METHOD=BAYES` so there will be two estimation methods (the code currently greps `#METH` in the `.lst` file) and one set of parameter estimates. 

The change in this PR is causing `model_summary()` to fail when running `METHOD=BAYES` in this way.

Reply:
@kylebaron Thanks for reporting.  Could you provide me with a sample model file that triggers the failure (either in a dedicated issue or offline)?

Reply:
Yes; I believe you will see this in the Bayes example project. I'll test and send you the link offline.

Reply:
@kylebaron thanks, and sorry for the breakage.